Source/Utils/WarpXUtil.cpp

```diff
@@ -394,6 +398,84 @@ parseStringtoInt(std::string str, std::string name)
     return ival;
 }
 
+int
+parseSignalNameToNumber(const std::string &str)
```

Comment:
Little AMReX convention that I wish there was a `clang-format` rule for:
```suggestion
parseSignalNameToNumber (const std::string &str)
```